genie-web/src/test/java/com/netflix/genie/web/controllers/JobRestControllerIntegrationTests.java

```diff
@@ -607,12 +902,88 @@ private void testForConflicts(
                 MockMvcRequestBuilders
                     .post(JOBS_API)
                     .contentType(MediaType.APPLICATION_JSON)
-                    .content(objectMapper.writeValueAsBytes(jobConflictRequest))
+                    .content(this.objectMapper.writeValueAsBytes(jobConflictRequest))
             )
             .andExpect(MockMvcResultMatchers.status().isConflict());
     }
 
     /**
+     * Test the job submit method for success twice to validate the file cache use.
+     *
+     * @throws Exception If there is a problem.
+     */
+    @Test
+    public void testSubmitJobMethodTwiceSuccess() throws Exception {
```

Comment:
@ajoymajumdar can you verify that this is actually needed? it doesn't seem to be doing anything to me other than slowing the build down